### DISCUSSION TITLE
extsort usage error

### DISCUSSION THREAD

**Discussion by datatraveller1** [dathere/qsv#2382] [Q&A] [answered]
I tried to use this command:
`qsv extsort --select 1,7 file.csv out.csv`

but I get: `usage error: extsort CSV mode requires an index`
Isn't `1,7` a proper index or am I doing something wrong?
Thank you!

**Comment by datatraveller1** [comment, marked as answer]
Ah ok, it seems I have to use the index command before:
`qsv index file.csv | qsv extsort --select 1,7 file.csv out.csv`

**Comment by ondohotola** [comment]
two separate commands, you don’t (have to) pipe

**Comment by datatraveller1** [comment]
Thank you, yes two commands are better readable than piping!
```
qsv index file.csv
qsv extsort --select 1,7 file.csv out.csv
```
